src/components/App.stories.mdx

```diff
@@ -0,0 +1,152 @@
+import { Meta, Canvas, Story, ArgsTable } from "@storybook/addon-docs";
+import { within, waitFor, waitForElementToBeRemoved } from "@storybook/testing-library";
+import { expect } from "@storybook/jest";
+import { BrowserRouter } from "react-router-dom";
+
+import App from "./App";
+
+<Meta
+  component={App}
+  argTypes={{
+    form: {
+      control: false,
+    },
+    languageSelectorTarget: {
+      control: false,
+    },
+  }}
+  args={{
+    "form.translationEnabled": true,
+    baseUrl: `${process.env.REACT_APP_BASE_API_URL}`,
+  }}
+  parameters={{
+    mockData: [
+      {
+        url: `${process.env.REACT_APP_BASE_API_URL}i18n/info`,
+        method: "GET",
+        status: 200,
+        response: {
+          languages: [
+            { code: "nl", name: "Nederlands" },
+            { code: "en", name: "English" },
+          ],
+          current: "nl",
+        },
+      },
+      {
+        url: `${process.env.REACT_APP_BASE_API_URL}i18n/language`,
+        method: "PUT",
+        status: 204,
+        response: {},
+      },
+    ],
+  }}
+/>
+
+export const Template = (args) => {
+  const form = {
+    uuid: "81a22589-abce-4147-a2a3-62e9a56685aa",
+    name: "Mocked form",
+    loginRequired: false,
+    translationEnabled: args["form.translationEnabled"],
+    loginOptions: [],
+    autoLoginAuthenticationBackend: "",
+    paymentRequired: false,
+    paymentOptions: [],
+    literals: {
+      previousText: { resolved: "Previous page", value: "" },
+      beginText: { resolved: "Begin form", value: "" },
+      changeText: { resolved: "Change", value: "" },
+      confirmText: { resolved: "Confirm", value: "" },
+    },
+    slug: "jest",
+    url: "http://127.0.0.1:8000/api/v2/forms/81a22589-abce-4147-a2a3-62e9a56685aa",
+    steps: [
+      {
+        uuid: "fc9f2889-b1e4-4045-b88c-acf1adf716cb",
+        slug: "step1",
+        formDefinition: "Stap 1",
+        index: 0,
+        literals: {
+          previousText: { resolved: "Previous page", value: "" },
+          saveText: { resolved: "Save current information", value: "" },
+          nextText: { resolved: "Next", value: "" },
+        },
+        url: "http://127.0.0.1:8000/api/v2/forms/81a22589-abce-4147-a2a3-62e9a56685aa/steps/fc9f2889-b1e4-4045-b88c-acf1adf716cb",
+      },
+    ],
+    showProgressIndicator: true,
+    maintenanceMode: false,
+    active: true,
+    explanationTemplate: "<p>Toelichtingssjabloon...</p>",
+    submissionAllowed: "yes",
+    requiredFieldsWithAsterisk: true,
+  };
+  return (
+    <BrowserRouter>
+      <App form={form} {...args} />
+    </BrowserRouter>
+  );
+};
+
+/*
+  FIXME: Why doesn't Storybook render these docs?
+  Uncaught TypeError: string is undefined
+*/
```

Comment:
Very disorienting to debug this... One thing I learned is that whitespace is significant.